smart-contracts/test/Lock/updateKeyPricing.js

```diff
@@ -95,14 +95,30 @@ contract('Lock / updateKeyPricing', accounts => {
       assert.equal(tokenAddressAfter, token.address)
     })
 
-    it('should allow the lock owner to switch from erc20 => eth', async () => {
+    it('should allow a LockManager to switch from erc20 => eth', async () => {
       await lock.updateKeyPricing(
         await lock.keyPrice.call(),
         Web3Utils.padLeft(0, 40)
       )
       assert.equal(await lock.tokenAddress.call(), 0)
     })
 
+    it('should allow a lock manager who is not the owner to make changes', async () => {
+      await lock.addLockManager(accounts[8], { from: lockOwner })
```

Comment:
Coverage is a little low on this feature, but it's kinda low risk since most of the work is from ZOS.  I guess it's alright, but we'll want to circle back and address that at some point.  e.g. renounceOwner, confirm only owner can add managers, confirm a manager can't add another manager, etc.

Some day I want to see us get to 100% coverage ;)

Reply:
I don't mind spending a bit more time adding tests here. Note that currrently, any lockManager can add lockManagers, and that only the lock owner is made a lockManager by default.